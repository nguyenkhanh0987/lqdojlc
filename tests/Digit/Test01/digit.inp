100
50802 67060
20882 43462
39371 66299
69887 92034
87457 98886
1944 22118
88035 88501
56541 91854
14798 91248
23676 72573
73579 95301
28940 39441
88130 93558
25344 98634
88643 89551
51219 70854
45954 86714
84908 95283
91066 93211
73150 86201
87660 97570
76039 81090
9139 64715
24445 29077
87730 97140
70901 95358
32690 45763
66205 75244
47299 77526
73726 96555
16716 68038
52160 78723
86539 97865
44971 82876
49509 74864
41974 55918
75287 82022
93238 97951
6126 81392
7312 8980
14051 91756
12437 97591
73423 86209
62738 81647
32654 76524
97463 97807
76359 98011
39036 63210
66346 86483
41081 81079
3516 52192
62466 73911
37253 49747
89041 97346
87005 99707
73951 77894
40433 49423
19051 37486
90882 98230
30863 40083
7284 68747
90170 97177
65282 65592
1541 96508
96063 99193
75809 82601
32757 88908
37750 46602
685 45682
69372 91743
84919 92847
65192 92735
47911 90268
65537 76923
89985 90004
62808 97783
45577 96956
68033 75171
19532 95471
2456 7673
71328 93975
37860 91760
1412 2988
49068 91501
68340 88960
8784 30565
60882 65902
20947 45816
8566 10225
26061 47692
22347 25118
1673 20028
57460 96772
62549 91147
35269 51481
63867 94992
68041 71278
77844 94950
18963 23408
30286 74593