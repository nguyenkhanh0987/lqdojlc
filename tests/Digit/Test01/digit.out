1141
3149
2377
871
370
4113
73
2099
7992
5470
865
1585
179
6234
99
1551
2959
358
5
568
299
403
7411
987
271
895
1598
562
2354
894
6380
1892
362
2974
1707
1326
416
97
9412
506
8148
8436
559
1158
4114
4
821
2330
1066
3181
7927
683
1744
199
507
218
1010
2772
136
1364
8380
79
21
10677
170
408
4707
1021
7492
938
331
1417
2899
687
0
1620
3291
393
6775
1191
866
4101
171
2659
993
4036
159
3354
496
3058
322
3993
2037
1518
2140
1504
319
631
612
4123